src/docs/development/tools/vs-code.md

```diff
@@ -232,6 +232,9 @@ The Flutter extension includes the following snippets:
 
 You can also define custom snippets by executing
 **Configure User Snippets** from the [Command Palette][].
+### Converting Stateful Widget to Stateless Widget
+Converting stateless to statefull can be used to spead up entering typical code structure.
+Foe converting to stateless to statefull just click on the bulb icon in vscode and then there you will see an option of converting stateless to statefull or vice versa. 
```

Comment:
- Foe -> For
- vscode -> VS Code
- ~there~
- .. **StatelessWidget to StatefulWidget** ..
- Sentence formation is not professional

Reply:
ok.
I will use the snippets sentence formation

Reply:
I have added changes you can see